symbolic-symcache/tests/snapshots/test_cache__functions_linux.snap

```diff
@@ -505,342 +512,460 @@ expression: FunctionsDebug(&symcache)
+            6304 _ZNSt12_Vector_baseIhN15google_breakpad16PageStdAllocatorIhEEEC4ERKS2_
+            6304 _ZNSt6vectorIhN15google_breakpad16PageStdAllocatorIhEEEC4ERKS2_
             631e _ZNSt6vectorIhN15google_breakpad16PageStdAllocatorIhEEE7reserveEm
-            62dc _ZNSt20_List_const_iteratorIN15google_breakpad12MappingEntryEEppEv
             6368 LogLine
+            63a2 LogAppend
+            63dc _ZN15google_breakpad13PageAllocator9GetNPagesEm
+            63dc sys_mmap
             64a4 DumpThreadStack
             64b1 LogAppend
-            64d0 LogAppend<long unsigned int>
             64d0 LogAppend
             64d0 LogAppend
+            64d0 LogAppend<long unsigned int>
             6625 LogAppend
-            6644 LogAppend<long unsigned int>
             6644 LogAppend
             6644 LogAppend
+            6644 LogAppend<long unsigned int>
             6799 LogAppend
-            67b8 LogAppend<long unsigned int>
             67b8 LogAppend
             67b8 LogAppend
+            67b8 LogAppend<long unsigned int>
             690d LogCommitLine
             6914 LogLine
-            695f LogAppend<long unsigned int>
             695f LogAppend
             695f LogAppend
+            695f LogAppend<long unsigned int>
             6978 LogAppend
             6af6 LogAppend
             6b18 _ZSt3minImERKT_S2_S2_
             6b2c LogAppend
-            6b4e LogAppend<unsigned char>
             6b4e LogAppend
             6b4e LogAppend
+            6b4e LogAppend<unsigned char>
             6ba0 LogCommitLine
             6ba7 LogLine
+            6c40 LogAppend
+            6c40 LogAppend
+            6c58 LogAppend
+            6c70 LogAppend
+            6c70 LogAppend
             6d45 LogLine
+            6d65 LogAppend
+            6d7d LogAppend
+            6d95 LogAppend
             6dc6 ~MicrodumpWriter
             6dd6 _ZN15google_breakpad17LinuxPtraceDumperD4Ev
             6e10 _ZNSt6vectorIhN15google_breakpad16PageStdAllocatorIhEEE15_M_range_insertIPKhEEvN9__gnu_cxx17__normal_iteratorIPhS3_EET_SB_St20forward_iterator_tag
-            6e24 _ZSt8distanceIPKhENSt15iterator_traitsIT_E15difference_typeES3_S3_
+            6e10 _ZNSt6vectorIhN15google_breakpad16PageStdAllocatorIhEEE15_M_range_insertIPKhEEvN9__gnu_cxx17__normal_iteratorIPhS3_EET_SB_St20forward_iterator_tag
+            6e10 _ZNSt6vectorIhN15google_breakpad16PageStdAllocatorIhEEE15_M_range_insertIPKhEEvN9__gnu_cxx17__normal_iteratorIPhS3_EET_SB_St20forward_iterator_tag
+            6e24 _ZSt10__distanceIPKhENSt15iterator_traitsIT_E15difference_typeES3_S3_St26random_access_iterator_tag
+            6e24 _ZSt10__distanceIPKhENSt15iterator_traitsIT_E15difference_typeES3_S3_St26random_access_iterator_tag
             6e24 _ZSt10__distanceIPKhENSt15iterator_traitsIT_E15difference_typeES3_S3_St26random_access_iterator_tag
+            6e24 _ZSt8distanceIPKhENSt15iterator_traitsIT_E15difference_typeES3_S3_
```

Comment:
These are interesting. The snapshot changes are only due to sorting, but nonetheless, do you know why exactly we have duplicates of these? What other properties do they differ in?